packages/tocco-ui/src/FormattedValue/typeFormatters/PhoneFormatter.spec.js

```diff
@@ -13,8 +13,8 @@ describe('tocco-ui', () => {
             .then(() => expect(wrapper.text()).to.equal('+41 44 388 60 00'))
         })
 
-        test('should show original string if its not a valid phonenumber', () => {
-          const invalidPhoneNumber = '+123 456'
+        test('should show original string if its not a valid phone number', () => {
+          const invalidPhoneNumber = '+41 4438860011111110'
```

Comment:
just curious , what's the motivation behind this change?

Reply:
The library formats the first 2-3 signs as prefix for all the possible areas (+1, +31, +3).

Reply:
As far as i know both are invalid phone numbers. the effect is the same, the component shows the original value unformatted?